add OpenTracing Shim usage (#1697)
open-telemetry/opentelemetry-java: opentracing_shim/README.md

[![Javadocs][javadoc-image]][javadoc-url]
# OpenTelemetry - OpenTracing Shim
A bridge layer from OpenTelemetry to the OpenTracing API.
It takes OpenTelemetry Tracer and expose it as an OpenTracing Tracer.

## Usage

> 1. Configure OpenTelemetry tracer

Reviewer: Can we mention we are talking about the global instance?

Author: done

> 1. Create OpenTracing tracer from OpenTelemetry tracer

Reviewer: Probably we should add an extra section on how to manually specify a `Tracer` with the rest of the components.

Author: done

[![Javadocs][javadoc-image]][javadoc-url]
# OpenTelemetry - OpenTracing Shim

> A bridge layer from OpenTelemetry to the OpenTracing API.
> It takes OpenTelemetry Tracer and expose it as an OpenTracing Tracer.

Reviewer 2: ```suggestion
The OpenTracing shim is a bridge layer from OpenTelemetry to the OpenTracing API.
It takes OpenTelemetry Tracer and exposes it as an implementation of an OpenTracing Tracer.
```

## Usage

> There are 2 ways to expose OpenTracing tracer: 

Reviewer 2: ```suggestion
There are 2 ways to expose an OpenTracing tracer: 
```

> 1. From global OpenTelemetry configuration 

Reviewer 2: ```suggestion
1. From the global OpenTelemetry configuration 
```

    ```java
    Tracer tracer = TraceShim.createTracerShim();
    ```

> 1. From provided `TracerProvider` and `CorrelationContextManager`

Reviewer 2: ```suggestion
1. From the provided `TracerProvider` and `CorrelationContextManager`:
```

    ```java
    Tracer tracer = TraceShim.createTracerShim(tracerProvider, contextManager);
    ```

> Optionally register tracer as the OpenTracing GlobalTracer

Reviewer 2: ```suggestion
Optionally register the tracer as the OpenTracing GlobalTracer:
```
